test/test_nn.py

```diff
@@ -1857,6 +1857,28 @@ def test_embedding_sparse_empty_tensor(self):
         self.assertTrue(embedding.weight.grad.is_sparse)
         self.assertEqual(embedding.weight.grad.shape, embedding.weight.shape)
 
+    def _test_embedding_dense_grad(self, dev):
+        embd = nn.Embedding(20, 20).to(dev)
+        weight = embd.weight
+
+        def fn_wrapper(dev):
+            def fn(weight):
+                inp = torch.tensor([[0, 1, 1, 2], [3, 5, 7, 11]], dtype=torch.long).to(dev)
+                return torch.nn.functional.embedding(inp, weight)
```

Comment:
can we add a few more tests with other parameters being tested as well, like `padding_idx`, `max_norm`, etc?

Reply:
There already are [tests](https://github.com/pytorch/pytorch/blob/0727eb87f25e804443d33e47edc8af999eb3803b/test/test_nn.py#L1882-L1964) for those parameters. Will add a test for double_backward in `padding_idx`.

Reply:
From what I see [here](https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1443-L1453), the normalization is independently applied before embedding. So I believe the test for max_norm should be independent like it is. Do let me know if I am missing something.

As for the `padding_idx`, I have extended the already present test to check for double backwards as well. Please review.

Thank You.